### ISSUE TITLE
Is there a way to change the font for Juno's integrated REPL console?

### ISSUE THREAD

**Issue by benninkrs** [JunoLab/atom-julia-client#413]

**Comment by pfitzseb** [comment]
It should always use the same font your editor uses (the one you set in Settings > Editor > Font Family).
Is that not the case for you?

**Comment by benninkrs** [comment]
Curious, if I set the font manually in Atom's editor settings, the Juno REPL adopts it.  But if I leave the option empty (default), the editor and REPL don't always have the same font.  It appears that the REPL starts with its own default, then defaults to whatever font was last manually specified.

Here's my system:
Windows 10
Atom 1.23.2
julia-client 0.6.7
Atom.jl 0.6.8
Juno.jl  0.4.0

**Comment by pfitzseb** [comment]
Right, if you don't explicitly set an editor font then the REPL uses xterm.js's default, whatever that might be. 
Tbh I don't think this is something I really want to find a workaround for since it's so easy to "fix" by users.
